impeller/renderer/render_target.cc

```diff
@@ -209,6 +209,10 @@ const std::optional<StencilAttachment>& RenderTarget::GetStencilAttachment()
   return stencil_;
 }
 
+ColorAttachment& RenderTarget::GetColorAttachment(size_t index) {
```

Comment:
The `index` is being ignored. You're always going to get 0u.

Reply:
On second thought, this seems like an unnecessary API change. You can `GetColorAttachments` to fetch the attachment, modify it as necessary, and then `SetColorAttachment`.

Reply:
Yeah good point, I would need to modify the map to not be const though

Reply:
SetColorAttachment should do the trick right?

Reply:
Yeah that will work!